pkgs/development/tools/heroku/default.nix

```
@@ -1,12 +1,19 @@
 { stdenv, lib, fetchurl, makeWrapper, nodejs }:
 
-stdenv.mkDerivation rec {
+let
+  # version and commit pairs can be found in the URLs at
+  # https://cli-assets.heroku.com/versions/heroku-linux-x64-tar-xz.json
+  version = "8.4.3";
+  commit = "8226515";
+  hash = "sha256-gbYKA9F3Ue4rcxRhJS4jbIFCEDV2Qy8xEsg0P09eopc=";
+in
+stdenv.mkDerivation {
   pname = "heroku";
-  version = "7.66.4";
+  inherit version;
 
   src = fetchurl {
```

```suggestion
  src = fetchzip {
```

I believe this supports `.ex`

---

remember to update the hash